ai-ml/jupyterhub/manifests/pv.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: efs-persist
+  namespace: ${namespace}
+spec:
+  capacity:
+    storage: 123Gi
+  accessModes:
+    - ReadWriteMany
+  nfs:
+    server: ${efs_id}.efs.${region}.amazonaws.com
```

Comment:
with the module you can get this directly https://github.com/terraform-aws-modules/terraform-aws-efs/blob/e59c4a960b5d2d92c0c5c99a07d2d9da49de2635/outputs.tf#L15

DNS suffix will not always be `amazonaws.com` depending on the region

Reply:
updated